client/jmx-adapter/src/test/java/org/jolokia/client/jmxadapter/JmxBridgeTest.java

```diff
@@ -510,11 +510,8 @@ public void testInvoke(ObjectName name, String operation, Object[] arguments)
     try {
       for (MBeanOperationInfo operationInfo : this.adapter.getMBeanInfo(name).getOperations()) {
         if (operationInfo.getName().equals(operation)
-            && operationInfo.getSignature().length == arguments.length) {
-          String[] signature = new String[operationInfo.getSignature().length];
-          for (int i = 0; i < signature.length; i++) {
-            signature[i] = operationInfo.getSignature()[i].getType();
-          }
+            && argumentCountIsCompatible(arguments, operationInfo.getSignature().length)) {
+          String[] signature = createSignature(operationInfo);
           Assert.assertEquals(
               this.adapter.invoke(name, operation, arguments, signature),
               nativeServer.invoke(name, operation, arguments, signature));
```

Comment:
Here we get test coverage and comparison that calling with a null signature works on both connections.